blocks/chat-cta/chat-cta.js

```diff
@@ -1,5 +1,27 @@
-const init = async (el) => {
-  const cta = el.children[0].children[0];
+export default async function init(el) {
+  const block = el;
+
+  if (el.dataset?.content) {
+    const contentUrl = el.dataset.content;
```

Comment:
You could pull this out to reuse the variable
```suggestion
  const contentUrl = el.dataset?.content;
  
  if (contentUrl) {
```